release/ray_release/byod/byod_horovod_master_test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# This script is used to build an extra layer on top of the base anyscale/ray image 
+# to run the horovod tests
+
+set -exo pipefail
+
+HOROVOD_WITH_GLOO=1 HOROVOD_WITHOUT_MPI=1 HOROVOD_WITHOUT_TENSORFLOW=1 HOROVOD_WITHOUT_MXNET=1 HOROVOD_WITH_PYTORCH=1 pip3 install -U git+https://github.com/horovod/horovod.git
```

Comment:
nit: could you add one or two slashes to make the line width smaller? maybe with in 80 chars?